lte/gateway/docker/readme.md

```diff
@@ -1,170 +1,29 @@
-# Containerization Deploy
+# Containerized AGW
 
-* Use the Base Cloudstrapper image with an expanded disk size (64G at least) to create Build Container
-    * If you want to use a stand-alone Ubuntu image, [Setup](https://docs.docker.com/engine/install/ubuntu/) Docker to run [locally](https://forums.docker.com/t/couldnt-connect-to-docker-daemon-at-http-docker-localhost-is-it-running/87257). Remember to re-login after local user got added to docker so that the new shell has all the right user provisions
-    * fb-magma-aws has ami-03bc7ef7f3b70f77b which is a base Cloudstrapper image with extended storage
-    * Run Cloudstrapper AGW deploy with input file as below (SDTI network already exists)
-    * devsrv-tokyo:~/magma-master #cat sdti-build1.yaml
+This folder contains container image definitions for AGW services.
 
-        ```
-        #Setting AGW AMI and Cloudstrapper AMI to expanded Cloudstrapper image Ubuntu to allow deploy
-        dirLocalInventory: ~/magma-master
-        awsAgwAmi: ami-03bc7ef7f3b70f77b
-        awsCloudstrapperAmi: ami-03bc7ef7f3b70f77b
-        awsAgwRegion: ap-northeast-1
-        keyHost: keyMagmaHostCharlie
-        idSite: SDTI
-        idGw: sdti-build1
-        awsInstanceType: t3.large
-        
-        devsrv-tokyo:~/magma-master/magma/experimental/cloudstrapper/playbooks #ansible-playbook —tags createGw agw-provision.yaml -e '@~/magma-master/sdti-build1.yaml'
-        ```
+## Preparing the host
 
-* Configure ssh config file to allow SSH access to CTR1
-* System information : Runs 5.4 kernel, has IP addresses configured on first and second interfaces
+The containerized AGW requires a specific setup on the host machine.
+The script ../agw_install_docker.sh can be used to configure a host
+accordingly.
```

Comment:
Which host your local machine?

Reply:
I thought it would. The script will run the [magma_deploy](https://github.com/magma/magma/blob/57820d4a20c21b94166f0f65aeb25a753e2948ff/lte/gateway/deploy/magma_docker.yml) playbook which sources the [magma_deploy](https://github.com/magma/magma/blob/20f437d03cce37b06ac38bf18724f3a7ef95817d/lte/gateway/deploy/roles/magma_deploy/tasks/main.yml#L128) and [agw_docker](https://github.com/magma/magma/blob/57820d4a20c21b94166f0f65aeb25a753e2948ff/lte/gateway/deploy/magma_docker.yml) roles. The magma_deploy role has a "_Install prebuilt openvswitch packages_" step and I hoped these prebuilt packages would have the correct patches.